### PR TITLE
Made qb-vehiclekeys fully translatable and added en.lua and es.lua

### PR DESCRIPTION
**Pull request by buddizer**
Describe Pull request
Made qb-vehiclekeys fully translatable and added en.lua and es.lua. Tested working.

Questions (please complete the following information):

Have you personally loaded this code into an updated qbcore project and checked all it's functionality? Yes
Does your code fit the style guidelines? Yes
Does your PR fit the contribution guidelines? Yes

### CODE REVIEW

**Review by ItsANoBrainer — APPROVED** [review]

**Comment by buddizer** [comment]
The linting report is saying args is not being used, I did not make any changes to that part of the code myself.

File: client/main.lua [hunk 1]
```diff
@@ -40,7 +40,7 @@ CreateThread(function()
                         if not isTakingKeys then
                             isTakingKeys = true
                             SetVehicleDoorsLocked(entering, 1)
-                            QBCore.Functions.Progressbar("steal_keys", "Taking keys from body...", 2500, false, false, {
+                            QBCore.Functions.Progressbar("steal_keys", Lang:t('info.taking_keys'), 2500, false, false, {
```
Comment (L1quidSn4ke) [inline client/main.lua:43]: Good Change, No Issues

File: client/main.lua [hunk 2]
```diff
@@ -70,7 +70,7 @@ CreateThread(function()
                     sleep = 5
 
                     local vehiclePos = GetOffsetFromEntityInWorldCoords(vehicle, 0.0, 1.0, 0.5)
-                    DrawText3D(vehiclePos.x, vehiclePos.y, vehiclePos.z, "~g~[H]~w~ - Search for Keys")
+                    DrawText3D(vehiclePos.x, vehiclePos.y, vehiclePos.z, Lang:t('info.search_keys'))
```
Comment (L1quidSn4ke) [inline client/main.lua:73]: Good Change, No Issues

File: client/main.lua [hunk 3]
```diff
@@ -122,7 +122,7 @@ end
 ---- Client Events ----
 -----------------------
 
-RegisterKeyMapping('togglelocks', 'Toggle Vehicle Locks', 'keyboard', 'L')
+RegisterKeyMapping('togglelocks', Lang:t('info.toggle_locks'), 'keyboard', 'L')
```
Comment (L1quidSn4ke) [inline client/main.lua:125]: Good Change, No Issues

File: client/main.lua [hunk 4]
```diff
@@ -192,7 +192,7 @@ RegisterNetEvent('qb-vehiclekeys:client:GiveKeys', function(id)
                 end
             end
         else
-            QBCore.Functions.Notify("You don't have keys to this vehicle.", 'error')
+            QBCore.Functions.Notify(Lang:t('error.no_keys'), 'error')
```
Comment (L1quidSn4ke) [inline client/main.lua:195]: Good Change, No Issues

File: client/main.lua [hunk 5]
```diff
@@ -217,7 +217,7 @@ function GiveKeys(id, plate)
     if distance < 1.5 and distance > 0.0 then
         TriggerServerEvent('qb-vehiclekeys:server:GiveVehicleKeys', id, plate)
     else
-        QBCore.Functions.Notify('There is nobody nearby to hand keys to.','error')
+        QBCore.Functions.Notify(Lang:t('error.nobody_nearby'),'error')
```
Comment (L1quidSn4ke) [inline client/main.lua:220]: Good Change, No Issues

File: client/main.lua [hunk 6]
```diff
@@ -285,10 +285,10 @@ function ToggleVehicleLocks(veh)
                 NetworkRequestControlOfEntity(veh)
                 if vehLockStatus == 1 then
                     SetVehicleDoorsLocked(veh, 2)
-                    QBCore.Functions.Notify("Vehicle locked!", "primary")
+                    QBCore.Functions.Notify(Lang:t('warning.vehicle_locked'), "primary")
```
Comment (L1quidSn4ke) [inline client/main.lua:288]: Good Change, No Issues

**Pull request closed without merging**